modules/gdscript/gdscript_parser.cpp

```diff
@@ -8774,7 +8774,7 @@ int GDScriptParser::get_completion_argument_index() {
 	return completion_argument;
 }
 
-int GDScriptParser::get_completion_identifier_is_function() {
+bool GDScriptParser::get_completion_identifier_is_function() {
```

Comment:
Why is this change here? I get that `bool` makes more sense, but it's not related to the other changes.

Reply:
Because it's a minor fix that I noticed while making this PR and it's easier than making a separate PR just for this one change, but I could do that if you want.

Reply:
At least do it in a different commit, so the blame is more precise.

Reply:
Done